docs: update docs around column aliasing (#5427)
confluentinc/ksql: docs/developer-guide/ksqldb-reference/select-push-query.md

Push a continuous stream of updates to the ksqlDB stream or table. The result of
this statement isn't persisted in a Kafka topic and is printed out only in
the console, or returned to the client. To stop a push query started in the CLI press Ctrl+C.

Execute a push query via the CLI or by sending an HTTP request to the ksqlDB REST API, and
the API sends back a chunked response of indefinite length.

> Push queries enable you to subscribe to changes, which enable

Author: I've changed this from:

> Push queries enable you to query a materialized view

As that would suggest you can only perform push queries on MVs.... which will suggest tables to most users, while ksqlDB supports push queries on streams too.

reacting to new information in real-time. They’re a good fit for asynchronous
application flows. For request/response flows, see [Pull Queries](select-pull-query.md).

Push queries can use all available SQL features, which can be useful when prototyping a

> persistent query or when running ad-hoc queries from the CLI. However, unlike persistent queries,

Reviewer: ```suggestion
persistent query or when running ad-hoc queries from the CLI. But unlike persistent queries,

```

> push queries are not shared. If multiple clients submit the same push query ksqlDB will compute

Reviewer: ```suggestion
push queries are not shared. If multiple clients submit the same push query, ksqlDB computes

```

independent results for each client.

!!! tip

>   If using push queries from an application, move all the heavy lifting into a persistent query

Reviewer: ```tip
  If you're using push queries from an application, move all the heavy lifting into a persistent query

```

#### WINDOW

!!! note

>   WINDOW clause can only be used if the `from_item` is a stream.

Reviewer: ```suggestion
  You can use the WINDOW clause only if the `from_item` is a stream.

```
